keyboards/void/voidhhkb_hotswap/keyboard.json

```
@@ -0,0 +1,96 @@
+{
+    "manufacturer": "James Wilson",
+    "keyboard_name": "voidhhkb_hotswap",
+    "maintainer": "JamesWilson1996",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["B12", "B13", "A3", "A4", "B14", "B15", "A8", "A5", "A9", "A10", "A6", "A7", "A15", "B3", "B4"],
+        "rows": ["B11", "B10", "B2", "B1", "B0"]
+    },
+    "processor": "STM32F072",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0002",
+        "vid": "0x4A57"
+    },
+    "layouts": {
+        "LAYOUT": {
```

This appears to match the 60_hhkb layout, and should be configured to use that. 
```suggestion
    "community_layouts": [ "60_hhkb" ],
    "layouts": {
        "LAYOUT_60_hhkb": {
```

---

Thanks. Implemented suggestion